evm/README.md

```diff
@@ -187,3 +187,97 @@ To get details of options provided in the command run:
 ```bash
 node evm/verify-contract.js --help
 ```
+
+## Combine Validator Signatures
+
+The following script is for emergency situations where we need to ask validators to manually sign gateway approvals. These validator signatures must be collected and stored within separate files in the signatures folder in the root directory. The script will then combine these signatures into a gateway batch and optionally execute this batch on the Axelar gateway contract on the specified chain.
+
+Ensure that the lcd endpoint has been provided for the specified environment in the chains config file under `axelar`:
+
+```json
+"axelar": {
+    "id": "Axelarnet",
+    "axelarId": "Axelarnet",
+    "rpc": "",
+    "lcd": "",
+    "grpc": "",
+    "tokenSymbol": "AXL"
+  }
+```
+
+Below are instructions on how to utilize this script:
+
+1. Construct the gateway batch data by passing in the payload hash gotten from running the `governance.js` script.
```

Comment:
Checked `governance.js` we are not encoding data in the format expected by `execute()`, check: 
- https://github.com/axelarnetwork/axelar-contract-deployments/blob/cc6c0ca5284ff17adf1b9116422f43833450745b/evm/governance.js#L192
- https://github.com/axelarnetwork/axelar-cgp-solidity/blob/968c8964061f594c80dd111887edb93c5069e51e/contracts/AxelarGateway.sol#L487C69-L487C106

Reply:
So how can we get payload hash from `governance.js`? @milapsheth @deanamiel